railties/lib/rails/generators/test_unit/authentication/templates/test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,31 @@
+require "test_helper"
+
+class SessionsControllerTest < ActionDispatch::IntegrationTest
+  test "new" do
+    get new_session_url
+    assert_response :success
+  end
+
+  test "create with valid credentials" do
+    post session_url, params: { email_address: "one@example.com", password: "password" }
+
+    assert_redirected_to root_url
+    assert parsed_cookies.signed[:session_id]
+  end
+
+  test "create with invalid credentials" do
+    post session_url, params: { email_address: "one@example.com", password: "wrong" }
+
+    assert_redirected_to new_session_url
+    assert_nil parsed_cookies.signed[:session_id]
```

Comment:
Why can't this just use the same assertion as in destroy?

Reply:
True, and `parsed_cookies` is probably not needed overall, since these tests only check for the presence of a value instead of a particular value.

I updated the PR accordingly. Note: I had to keep `assert_nil` here vs `assert_empty` in the destroy for the tests to pass when trying it out on a local test app.